### PR TITLE
Pebble shotcanister material fix

### LINKED ISSUE #38705: Pebble shotcanister has the wrong material (Steel)
# Describe the bug
The Pebble Shotcanister is shown as Material: Steel 

# Steps To Reproduce

Try to craft it, check the Material.

# Expected behavior

Being Material: Stone

# Screenshots

grafik

# Versions and configuration
- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.D-13021-g0840d86 [64-bit]
- Graphics Version: Tiles
- Mods loaded: [
    Dark Days Ahead [dda],
    SpeedyDex [speedydex],
    Nonperishable Overhaul [Nonperishable_Overhaul],
    No Fungal Monsters [No_Fungi],
    Graphical Overmap [Graphical_Overmap]
]

# Additional context
Change in data\json\items\ammo\shotcanister.json line 33
` 
    "material": [ "steel" ],
`
to

`     "material": [ "rock" ],
`

### PR DESCRIPTION
**Pull request by Funguss**
#### Summary
```SUMMARY: Bugfixes "Give pebble shotcanister material stone"```

#### Purpose of change
Fixes #38705 : Pebble shotcanister has the wrong material.

#### Describe the solution
Simple JSON edit from "steel" to "stone".

#### Testing
Spawned and checked, no issues. Was probably a bit overboard to check this, but there we go.

### CODE REVIEW

**Comment by jkraybill** [comment]
It's actually made of paper, so I think that should be the material?

**Comment by Rail-Runner** [comment]
Probably need to specify both stone and paper as valid materials, then.

**Comment by Lamandus** [comment]
then it should be for every shotcanister. Because all shotcanister are missing "paper". But either way, Steel is wrong.

**Comment by Funguss** [comment]
~~Paper containing pebbles, no? So paper and stone methinks. Bang on. ~~
Wrote that last night and failed to send it. Oh well. Working on this but it's turning into a nightmare. Adding paper meant they were combustible so I had to ensure they dropped the correct amount of resources. In doing that I found out the recipes were as wrong as they looked, so I updated them. However, there's still a conservation of mass issue with bearings as the recipe for those can be made with one scrap, turned into shotcanisters and burned. Each step weight just appears, and when burning the shotcanister you get several times more weight than you put in. 1 scrap = 50 bearings = 50 shotcanisters = 25 scrap. Last night I quit after accidentally worsening the issue (50 shotcanisters = 50 scrap).  I'll get back to it later today, but I'm pretty sure I can commit the changes I have and put bearing rationalization in a separate PR, the numbers should work then.

File: data/json/items/ammo/shotcanister.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "symbol": "=",
     "color": "green",
     "description": "This small paper canister resembling a crude shotshell is filled to the brim with tiny pebbles.  Damage is pathetic but flight is much more stable than scrap",
```
Comment (Night-Pryanik) [inline data/json/items/ammo/shotcanister.json]:
```suggestion
    "description": "This small paper canister resembling a crude shotshell is filled to the brim with tiny pebbles.  Damage is pathetic but flight is much more stable than scrap.",
```

**Pull request merged**